src/openapi.json

```diff
@@ -1544,6 +1600,121 @@
 				}
 			}
 		},
+		"/baseFields/{baseFieldId}/localizations": {
+			"get": {
+				"operationId": "getBaseFieldLocalizationsByBaseFieldId",
+				"summary": "Gets a list of base fields localizations for a given basefield.",
+				"tags": ["Base Fields"],
+				"security": [
+					{
+						"auth": []
+					}
+				],
+				"parameters": [
+					{
+						"name": "baseFieldId",
+						"description": "The PDC-generated ID of a base field.",
+						"in": "path",
+						"required": true,
+						"schema": {
+							"type": "integer"
+						}
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "All base field localizations for a given base field in the PDC.",
+						"content": {
+							"application/json": {
+								"schema": {
+									"$ref": "#/components/schemas/BaseFieldLocalizationBundle"
+								}
+							}
+						}
+					},
+					"404": {
+						"description": "The base field could not be found."
+					}
+				}
+			}
+		},
+		"/baseFields/{baseFieldId}/localizations/{language}": {
+			"put": {
+				"operationId": "upsertBaseFieldLocalization",
+				"summary": "Create or update a base field localization.",
```

Comment:
We're using the { whatever the linguistic term is } form of verbs in summaries.

So: "Creates or updates" rather than "Create or update"